src/app/content/studyGuides/hooks/receiveBook.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { HighlightsSummary } from '@openstax/highlighter/dist/api';
+import createTestServices from '../../../../test/createTestServices';
+import createTestStore from '../../../../test/createTestStore';
+import { book } from '../../../../test/mocks/archiveLoader';
+import { mockCmsBook } from '../../../../test/mocks/osWebLoader';
+import { resetModules } from '../../../../test/utils';
+import { MiddlewareAPI, Store } from '../../../types';
+import { receiveBook } from '../../actions';
+import { formatBookData } from '../../utils';
+import { receiveStudyGuides } from '../actions';
+
+const mockConfig = {BOOKS: {
+ [book.id]: {defaultVersion: book.version},
+} as {[key: string]: {defaultVersion: string}}};
+
+jest.doMock('../../../../config', () => mockConfig);
```

Comment:
I'm not sure if this is necessary here. I don't think the current tests depend on it.